### DISCUSSION TITLE
Get list of pages by filtering on createDate greater than in graphql?

### DISCUSSION THREAD

**Discussion by andree-rehnberg** [requarks/wiki#3181] [Help / Questions]
Is it possible to get a list of pages based on the pages createDates being greater than the date string I send a request?
Ideally I'd like to do something like the following:
```
{
  pages {
    list(filter: { createDate_gt: "2021-01-05T08:35:21.485Z" }) {
      id
    }
}
```
Is there some query I can do that gets me the same result?

**Comment by jackadam1981** [comment]
I'm looking for a way to update a date greater than a specified date.

query {
  pages {
    list{
        id
        updatedAt
    }
  }
}

I found the search query, but I didn't find the specific usage.
query {
  pages {
    search(query: ""){
      results{
        id
        title
      }
    }
  }
}
